common/lib/xmodule/xmodule/course_module.py

```diff
@@ -740,6 +740,16 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    enable_unproctored_exams = Boolean(
+        display_name=_("Enable Unproctored Exams"),
+        help=_(
+            "Enter true or false. If this value is true, unproctored exams are enabled in your course. "
```

Comment:
"Enter true or false. If this value is true, learners have the option to take proctored exams without proctoring."